tests/test_server/fixtures/factories/user.py

```diff
@@ -27,7 +27,7 @@ async def user(
 ) -> AsyncGenerator[User, None]:
     params = request.param
     item = user_factory(**params)
-
+    del item.id
```

Comment:
what is the purpose of deleting item.id?

Reply:
Sometimes you don't care about item.id and it can be random(from facrotry), and sometimes(in most cases) you need an id which will be generate by database.